### PR TITLE
update json file

### PR DESCRIPTION
**Pull request by ticoann**
1. change the name outputLFN/PFN to outputLFNs/PFNs since it is changed to list
2. add fallbackFiles, skippedFiles to fileArrayRef
3. remove missing empty dictionary from production sample

### CODE REVIEW

**Comment by ticoann** [comment]
@vkuznet, @yuyiguo Please review.

**Comment by ticoann** [comment]
Yuyi, also convert function is ready in https://github.com/dmwm/WMCore/pull/6614 

https://github.com/dmwm/WMCore/pull/6614/files#diff-e7002dd3cf6e272f53e7dbdeea251a59R193

File: test/data/samples/fwjr_production.py [hunk 1]
```diff
@@ -8,11 +8,11 @@
                'root://eoscms.cern.ch//eos/cms/store/data/Run2011A/Cosmics/RAW/v1/000/160/960/E8099605-8853-E011-A848-0030487A18F2.root',
                '/store/unmerged/CMSSW_7_0_0_pre11/Cosmics/ALCARECO/DtCalib-RECOCOSD_TaskChain_Data_pile_up_test-v1/00000/ECCFE421-08CB-E511-9F4C-02163E017804.root',
                'root://eoscms.cern.ch//eos/cms/store/unmerged/CMSSW_7_0_0_pre11/Cosmics/ALCARECO/DtCalib-RECOCOSD_TaskChain_Data_pile_up_test-v1/00000/ECCFE421-08CB-E511-9F4C-02163E017804.root',
-               ],
+               'blohblah'],
```
Comment (vkuznet) [inline test/data/samples/fwjr_production.py]: why do we need that? We already have a list of strings, so in my opinion there is no need for additional string in a list

File: test/data/samples/fwjr_production.py [hunk 2]
```diff
@@ -92,8 +92,8 @@
               'status': 0,
               'stop': 1454569736}
             ],
-'fallbackFiles': ['blahblah'],
-'skippedFiles': ['blahblah'],
+'fallbackFiles': [4],
```
Comment (vkuznet) [inline test/data/samples/fwjr_production.py:95]: I think I understand now the usage of "blohblah" in fileArrays, it refers to fallbackFiles, right? If so can we modify the string to make it more clear, e.g. /lfn/fallbackfiles.root

Comment (ticoann) [inline test/data/samples/fwjr_production.py:95]: updated now.

**Comment by yuyiguo** [comment]
@ticoann @vkuznet 

Thanks Seangchan, It looked good to me. I am going to merge it if no more comments. Is the converting code updated?
Yuyi

**Comment by vkuznet** [comment]
Yuyi,
I already merged the PR.
V.

**Comment by ticoann** [comment]
@yuyiguo Yes, converting code is up to date.

**Pull request merged**